.github/workflows/build.yml

```diff
@@ -109,13 +109,15 @@ jobs:
           git config --global user.email "github-ci-build@phing.info"
           git config --global user.name "Phing Github Action"
           echo "=== RUN TESTS ==="
-          ../bin/phing -debug -Dtests.codecoverage=true -listener "phing.listener.StatisticsListener"
+          XDEBUG_MODE=coverage ../bin/phing -debug -Dtests.codecoverage=true -listener "phing.listener.StatisticsListener"
 
       - name: Transfer coverage
         if: github.repository_owner == 'phingofficial'
+        with:
+          php-version: '7.4'
         run: |
```

Comment:
@jawira you cannot have both `with` and `run` in one step.
I would suggest to go by:
```yaml

  coverage:
    runs-on: ubuntu-latest
    needs:
      - static_code_analysis
    steps:
      - name: Checkout repository
        uses: actions/checkout@v2

      - name: Setup PHP
        uses: shivammathur/setup-php@v2
        with:
          php-version: '7.4'
          ini-values: xdebug.mode=coverage

      - name: Get composer cache directory
        id: composer-cache
        run: echo "::set-output name=dir::$(composer config cache-files-dir)"

      - name: Cache dependencies
        uses: actions/cache@v2
        with:
          path: ${{ steps.composer-cache.outputs.dir }}
          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
          restore-keys: ${{ runner.os }}-composer-

      - name: Test with coverage
        working-directory: test
        run: |
          echo "=== SETTING GIT IDENTITY ==="
          git config --global user.email "github-ci-build@phing.info"
          git config --global user.name "Phing Github Action"
          composer install --no-interaction --no-progress --ansi
          echo "=== RUN TESTS ==="
          ../bin/phing -debug -Dtests.codecoverage=true -listener "phing.listener.StatisticsListener"

      - name: Transfer coverage
        if: github.repository_owner == 'phingofficial'
        run: |
          bash <(curl -s https://codecov.io/bash) -f ./test/reports/clover-coverage.xml
          wget https://scrutinizer-ci.com/ocular.phar && php ocular.phar code-coverage:upload --format=php-clover ./test/reports/clover-coverage.xml

```